python/ray/tune/trial.py

```diff
@@ -301,6 +305,18 @@ def __init__(self,
         self.num_failures = 0
 
         self.custom_trial_name = None
+        self.results_since_checkpoint = []
+        self.past_avg = float("-inf")
+        self.prefix = {
+            "best/": {
```

Comment:
as far as I can see the only usage is in os.path.join, so "best" would work as well (remove /)

Reply:
true, removing